Xwt.XamMac/Xwt.Mac/ImageBuilderBackendHandler.cs

```diff
@@ -78,12 +78,14 @@ public override object CreateContext (object backend)
 		public override object CreateImage (object backend)
 		{
 			var gc = (CGContextBackend)backend;
-			var img = new NSImage (((CGBitmapContext)gc.Context).ToImage (), gc.Size);
-			var imageData = img.AsTiff ();
-			var imageRep = (NSBitmapImageRep) NSBitmapImageRep.ImageRepFromData (imageData);
-			var im = new NSImage ();
-			im.AddRepresentation (imageRep);
-			return im;
+			using (var img = new NSImage(((CGBitmapContext)gc.Context).ToImage(), gc.Size))
+			{
+				var imageData = img.AsTiff();
```

Comment:
Does this need to be with a using statement. I can see you added a using for it in ImageHandler.

Reply:
Wasn't sure on the copying semantics of `NSBitmapImageRep. ImageRepFromData`.

If it's just storing the value we are passing, then disposing it would probably trigger another object construction later, as we'll query ImageRepresentations.

If it's creating a copy of our data, then yeah, it should be added in a using. Will test it shortly.

Reply:
It's definitely a copy!

Reply:
Fixed!